integration-test/test.sh

```diff
@@ -4,7 +4,7 @@ IFS=$'\n\t'
 
 # these flighting values should match Makefile
 readonly TEST_SUBS_AZURE="c3dfd792-49a4-4b06-80fc-6fc6d06c4742"
-readonly TEST_REGION_AZURE="Brazil South"
+readonly TEST_REGION_AZURE="Central US EUAP"
```

Comment:
It was for testing when I was trying to mitigate the issue, and was not intended for anything long term.  The previous maintainer's stance was to publish to Brazil South, and test there.  If everything looked good, he would publish it more regions.

The new goal is to use EDP instead, and EDP prefers South Central US.  I have updated the region once again as a result.